### PR TITLE
Feature/214 fullscreen video

### PR DESCRIPTION
**Pull request by Mortadha-abderrhim**
This PR allows seeing the video in fullscreen.
- A custom MediaController that supports fullscreen mode is created.
- The fullscreen mode is shown in a new activity.
- The video is stored locally when the experience is accessed and is deleted when exited (keeping it would take much storage space).
- Some design improvements were made.

### CODE REVIEW

**Review by amomo25 — COMMENTED** [review]
Great PR, I just found small issues in the code, but the new design looks really good, and the fact that the video is cached makes the viewing of videos more pleasant.

File: app/src/main/java/com/pindex/main/media/video/FullscreenMediaController.kt [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+/**
+ * MediaController showing a fullScreen button
+ * when the button is clicked a fullscreen video activity is started
+ */
+class FullscreenMediaController(
+    activity: FragmentActivity?,
+    private val isFullscreen: Boolean,
+    private val calling: AppCompatActivity,
+    private val videoUrl: String?,
+    private val fragment: VideoBlockFragment
```
[the remark this answers is not in the document]
Comment (Mortadha-abderrhim) [reply app/src/main/java/com/pindex/main/media/video/FullscreenMediaController.kt:24]: Nice remark

**Review by amomo25 — DISMISSED** [review]

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -50,6 +50,7 @@
 
         <activity
             android:name=".ui.activities.ExperienceActivity"
+            android:launchMode="singleTask"
             android:parentActivityName=".ui.activities.HomeActivity" />
```
Comment (ToufiPF) [inline app/src/main/AndroidManifest.xml:54]: Are we sure we want the singleTask here? Doesn't it make the Activity a singleton or something? That doesn't really sounds like what we want from ExperienceActivity, does it?

Comment (Mortadha-abderrhim) [reply app/src/main/AndroidManifest.xml:54]: You are right, It was something that I added a while ago when trying to resolve the reloading issue without cache.. I will delete that.

**Comment by lucasrlt** [comment]
Do you think there is a way to fix the bug that when the video is in full screen, the controls overlap the action bar at the bottom of the Android phone ?

**Review by amomo25 — DISMISSED** [review]

**Review by amomo25 — APPROVED** [review]

**Review by amomo25 — APPROVED** [review]

**Review by lucasrlt — APPROVED** [review]
Nice fix with the overlap for the controls bar ! :+1: 

Great PR, the video looks way better now

**Pull request merged**